tests/mpd/protocol/test_idle.py

```diff
@@ -102,6 +108,22 @@ def test_idle_player_then_event_player(self):
         self.assertOnceInResponse('changed: player')
         self.assertOnceInResponse('OK')
 
+    def test_idle_then_output(self):
+        self.send_request('idle')
+        self.idle_event('player')
+        self.assertNoSubscriptions()
+        self.assertNoEvents()
+        self.assertOnceInResponse('changed: player')
+        self.assertOnceInResponse('OK')
+
```

Comment:
A copy-paste error here, should be "output" of course. Fixed that in latest commit.